dev/run-tests-jenkins.py

```diff
@@ -182,6 +182,11 @@ def main():
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop2.7"
     if "test-hadoop3.2" in ghprb_pull_title:
         os.environ["AMPLAB_JENKINS_BUILD_PROFILE"] = "hadoop3.2"
+    # Switch the Hive profile based on the PR title:
+    if "test-hive1.2" in ghprb_pull_title:
+        os.environ["AMPLAB_JENKINS_BUILD_HIVE_PROFILE"] = "hive1.2"
+    if "test-hive2.3" in ghprb_pull_title:
+        os.environ["AMPLAB_JENKINS_BUILD_HIVE_PROFILE"] = "hive2.3"
```

Comment:
cc @shaneknapp, maybe we can use it this environment variable (?) later when we consider setting the Jenkins jobs for Hive 1.2/2.3 + Hadoop 2.7/3.2 + JDK 8/11 combinations.

The env name looks a bit odd given `AMPLAB_JENKINS_BUILD_PROFILE`. We might have to rename it later.

Reply:
sorry, i missed this over the november holiday.  yes, i've had a desire for years to rename the `AMPLAB_*` variables as the amplab project ended in december 2016.  :)

i'll do this after the 3.0 cut.  no time like the present!